src/main/webapp/app/exam/manage/exam-management.component.html

```diff
@@ -1 +1,119 @@
-TODO
+<div>
+    <div class="d-flex">
+        <h4 id="course-page-heading" jhiTranslate="artemisApp.examManagement.title">Exams</h4>
+        <div class="ml-auto d-flex">
+            <button jhiOrionFilter class="btn btn-primary jh-create-entity create-exam" *jhiHasAnyAuthority="['ROLE_ADMIN', 'ROLE_INSTRUCTOR']" [routerLink]="['new']">
```

Comment:
More a question as an comment since I stumbled across it too: Do we need `jhiOrionFilter` and corresponding attributes for the exam management?

Reply:
I'm not sure, i added it as it was present in the other similar components such as the course-management component

Reply:
This is a specific filter for Orion. It specifies that the button should be shown in Orion. I think for now we can leave this out for all exam related functionalities